src/style_context.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2013-2020, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use glib::object::IsA;
+use pango::FontDescription;
+use StateFlags;
+use StyleContext;
+use StyleContextExt;
+
+pub trait StyleContextExtManual: 'static {
+    fn get_font(&self, state: StateFlags) -> Option<FontDescription>;
+}
+
+impl<O: IsA<StyleContext>> StyleContextExtManual for O {
+    fn get_font(&self, state: StateFlags) -> Option<FontDescription> {
```

Comment:
Can this ever be `None`?

Reply:
In normal case is not, want me add one more `except`?

Reply:
If it's a bug if there is no font description then yes

Reply:
Old https://developer.gnome.org/gtk3/stable/GtkStyleContext.html#gtk-style-context-get-font don't says that it can be null